### PR TITLE
rename na klasach dla porzadku - converet, savetofilewithnameandtext

### PR DESCRIPTION
**Pull request by postek**
Nic specjalnego, zmiana nazwy dwoch klasek dla porzadku

### CODE REVIEW

File: GameOfLife/checkbuttonpressed.cpp [hunk 1]
```diff
@@ -1,23 +1,50 @@
 #include "checkbuttonpressed.h"
 
+bool CheckButtonPressed::getSaveToFileStatus()
+{
+    return saveToFileStatus;
+}
+
```
Comment (piotrgajow) [inline GameOfLife/checkbuttonpressed.cpp]: If this PR contains renames, why these changes are added?

File: GameOfLife/checkbuttonpressed.cpp [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
 {
-   loopStatus = false;
+    loopStatus = false;
+    saveToFileStatus = false;
+    howManySec = 1;
+}
+
+int CheckButtonPressed::getHowManySec()
+{
+    return howManySec;
```
Comment (piotrgajow) [inline GameOfLife/checkbuttonpressed.cpp]: If this PR contains renames, why these changes are added?

File: GameOfLife/checkbuttonpressed.cpp [hunk 3]
```diff
@@ -1,23 +1,50 @@
...
+    }
+    else if(GetAsyncKeyState(VK_SUBTRACT))
+    {
+        howManySec/=2;
+        std::cout<<howManySec;
+    }
+    else if(GetAsyncKeyState(VK_CONTROL) && GetAsyncKeyState(0x53))
+    {
+        saveToFileStatus = true;
+    }
```
Comment (piotrgajow) [inline GameOfLife/checkbuttonpressed.cpp]: If this PR contains renames, why these changes are added?

File: GameOfLife/displaygameboardonthecommandline.cpp [hunk 4]
```diff
@@ -1,15 +1,18 @@
 #include "displaygameboardonthecommandline.h"
 
 
-
+int DisplayGameBoardOnTheCommandLine::getCoutnerOfIteration()
+{
+    return coutnerOfIteration;
+}
```
Comment (piotrgajow) [inline GameOfLife/displaygameboardonthecommandline.cpp]: If this PR contains renames, why these changes are added?

File: GameOfLife/displaygameboardonthecommandline.cpp [hunk 5]
```diff
@@ -1,15 +1,18 @@
...
 {
-
+    coutnerOfIteration = 0;
 }
 
 void DisplayGameBoardOnTheCommandLine::displayGameBoardOnTheCommandLine(std::string stringConvertedFromBoard)
 {
         std::cout<<stringConvertedFromBoard;
-        std::cout<<"--------------------------"<<std::endl;
+        std::cout<<"---------------------------- "<<++coutnerOfIteration<<std::endl;
```
Comment (piotrgajow) [inline GameOfLife/displaygameboardonthecommandline.cpp]: If this PR contains renames, why these changes are added?

File: GameOfLife/displaygameboardonthecommandline.h [hunk 6]
```diff
@@ -5,9 +5,12 @@
 
 class DisplayGameBoardOnTheCommandLine
 {
+private:
+    int coutnerOfIteration;
```
Comment (piotrgajow) [inline GameOfLife/displaygameboardonthecommandline.h]: Typo

File: GameOfLife/displaygameboardonthecommandline.h [hunk 7]
```diff
@@ -5,9 +5,12 @@
...
 public:
     DisplayGameBoardOnTheCommandLine();
     void displayGameBoardOnTheCommandLine(std::string stringConvertedFromBoard);
+    int getCoutnerOfIteration();
```
Comment (piotrgajow) [inline GameOfLife/displaygameboardonthecommandline.h]: Typo

File: GameOfLife/systemcomand.h [hunk 8]
```diff
@@ -0,0 +1,14 @@
+#ifndef SYSTEMCOMAND_H
+#define SYSTEMCOMAND_H
+#include "checkbuttonpressed.h"
+
+class systemComand
```
Comment (piotrgajow) [inline GameOfLife/systemcomand.h]: Class names should start with a capital letter

File: GameOfLife/checkbuttonpressed.cpp (hunk 1 above)
Comment (postek) [inline GameOfLife/checkbuttonpressed.cpp]: Just after creating PR it was only reaname. The work was still going on, and unfortunatelly next commits were added to this PR. We need solution on this case like creating additional remote branch after PR, so new chenges will not be added to previous PR. Sorry my fault, I should have predicted this, next commits will be pushed to another branch.

Greetings!

**Comment by postek** [comment]
Just after creating PR it was only reaname. The work was still going on, and unfortunatelly next commits were added to this PR. We need solution on this case like creating additional remote branch after PR, so new chenges will not be added to previous PR. Sorry my fault, I should have predicted this, next commits will be pushed to another branch. Since now ;P

Greetings!

**Comment by piotrgajow** [comment]
Then please fix the typos :)
